atomic_reactor/plugins/pre_koji.py

```diff
@@ -45,7 +45,17 @@ def run(self):
             self.log.error("provided target '%s' doesn't exist", self.target)
             raise RuntimeError("Provided target '%s' doesn't exist!" % self.target)
         tag_info = self.xmlrpc.getTag(target_info['build_tag_name'])
+
+        if not tag_info or 'name' not in tag_info:
```

Comment:
[Koji source](https://github.com/koji-project/koji/blob/master/util/kojira#L59) suggests it is. I experimented a bit on stage server and it seems it either returns `None` or a correct dict - this is what the tests are checking too.